xtests/tst-pam_motd2.c

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
To avoid code duplication, please just have a single tst-pam_motd.c which would take the service name as parameter.

Reply:
I did this and created a test file `tst-pam_motd.sh`, which then calls the individual `tst-pam_motd{1,2,3}.sh` which each specify the service name. Is this alright to do from the perspective of the testing structure? I can have all the tests done in the one `tst-pam_motd.sh` file as well

Reply:
Do as you prefer.